shared/types/form/form_issue.ts

```diff
@@ -0,0 +1,12 @@
+import { FormDto } from './form'
+
+/**
+ * Typing for individual form issue
+ */
+export type FormIssueBase = {
+  formId: FormDto['_id']
+  issue: string
+  email?: string
+  created?: Date
```

Comment:
Curious, why would created be an optional field?

Reply:
I think this is a common problem for creation, where the dataset we supply does not contain a `created` date (that gets generated by mongoose), but thereafter all reads will have a `created` field. Ideally we'd have a better way to declare the create/read distinction 🤔